include/mvd/mvd_generic.hpp

```diff
@@ -42,6 +44,42 @@ inline MVDFile* open_mvd(const std::string & filename) {
     return new MVD3::MVD3File(filename);
 }
 
+///
+/// \brief open
+///
+/// Opens a file with the associated class. A basic check for file type is
+/// performed, but no guarantees about file correctness are made.
+///
+/// \param filename the path of the file to open
+/// \return a shared pointer to a mvd::File object
+///
+inline std::shared_ptr<File> open(const std::string& filename) {
+    const std::string mvd3 = ".mvd3";
+
+    std::cout << filename << "\n";
```

Comment:
I am not sure this `std::cout << ...` is wanted.

Reply:
Ooops… yes